pkg/psmdb/backup/pbm.go

```diff
@@ -200,3 +200,12 @@ func (b *PBM) HasLocks(predicates ...LockHeaderPredicate) (bool, error) {
 
 	return false, nil
 }
+
+func (b *PBM) GetLastPITRChunk() (*pbm.PITRChunk, error) {
+	masterNode, err := b.C.GetNodeInfo()
+	if err != nil {
+		return nil, errors.Wrap(err, "getting master node")
+	}
+
+	return b.C.PITRLastChunkMeta(masterNode.SetName)
```

Comment:
1) it is wrong to get last chunk from database itself. we should restore to the latest chunk on the storage.
again: think about case `clusterA makes backups to storageA, clusterB makes backups to storageB.
now we need to restore PITR backup from storageA to clusterB to the latest backed up chunk`

2) `master` terminology can be sensitive, please keep using netral terminology here. e.g.: `nodeInfo`